src/mouse_tracking/cli/infer.py

```diff
@@ -6,6 +6,8 @@
 import click
 import typer
 
+# from mouse_tracking.tfs_inference import infer_arena_corner_model as infer_tfs
```

Comment:
I'm currently having issues with tensorflow. This will get added back when I can figure that out.